redshift_sqlalchemy/dialect.py

```diff
@@ -700,70 +700,189 @@ def _get_all_constraint_info(self, connection, **kw):
         return self._all_constraints
 
 
+# At the time of this implementation, no specification for a session token was
+# found. After looking at a few session tokens they appear to be the same as
+# the aws_secret_access_key pattern, but much longer. An example token can be
+# found here:
+#   http://docs.aws.amazon.com/STS/latest/APIReference/API_GetSessionToken.html
+# The regexs for access keys can be found here:
+#     http://blogs.aws.amazon.com/security/blog/tag/key+rotation
+creds_rx = re.compile(r"""
+    ^aws_access_key_id=[A-Z0-9]{20};
+    aws_secret_access_key=[A-Za-z0-9/+=]{40}
+    (?:;token=[A-Za-z0-9/+=]+)?$
+""", re.VERBOSE)
```

Comment:
Constants should be named in all caps, and I'd prefer `RE` or `REGEX` to `RX` unless this is conforming to a standard I'm not aware of (`CREDS_RE`).